### DISCUSSION TITLE
Started getting GHA successful workflow web notifications today?

### DISCUSSION THREAD

**Discussion by kingthorin** [community/community#28377] [Actions] [answered]
I started getting web notifications on successful GHA workflow runs today. Was that intentional? Can I turn that off somewhere?

![image](https://user-images.githubusercontent.com/7570458/182926177-6a9ce1d0-9127-49a3-99a4-e526976faf59.png)
![image](https://user-images.githubusercontent.com/7570458/182926298-3f010541-7dd7-443a-a1f5-039741edf9a6.png)

I haven't gotten these in the past. Normally just failures and cancelled (due to concurrency groups).

**Comment by airtower-luna** [comment, marked as answer]
#28367 looks like you weren't the only one who got those, but someone in that discussion found a way to turn it off again.

I personally haven't gotten unexpected notification (and just checked my settings to be sure), and with the weird way of turning it off again I suspect it's a bug. :sweat_smile:
